src/irclog2html/logs2html.py

```diff
@@ -211,6 +211,9 @@ def main(argv=sys.argv):
     parser.add_option('-g', '--glob-pattern', dest="pattern", default="*.log",
                       help="glob pattern that finds log files to be processed"
                       " (default: *.log)")
+    parser.add_option('-o', '--output-dir', dest="output_dir",
+                      help="destination output directory"
+                           " (default: current)")
```

Comment:
I believe the default should be "Same as input directory".

Reply:
I think to fix the failing build all you have to do is add a keyword argument `default=None` here.

Reply:
(You can drop the `dest="output_dir"` bit, since argparse automatically derives that name from `--output-dir`, but it's harmless.)